client/src/margo_client.c

```diff
@@ -290,6 +290,7 @@ int invoke_client_mount_rpc(void)
         LOGWARN("mismatch on app_id - using %d, server returned %d",
                 unifyfs_app_id, srvr_app_id);
     }
+    LOGERR("My client id is %d", unifyfs_client_id);
```

Comment:
Should this be a  `LOGDBG`?

Reply:
yes, it should. looks like I missed converting one back to LOGDBG. I'll update and repush.